docs/upgrade/upgrade-enterprise/index.rst

```diff
@@ -6,42 +6,44 @@ Upgrade Scylla Enterprise
 
 .. panel-box::
-  :title: Upgrade Scylla Enterprise
+  :title: Upgrade ScyllaDB Enterprise
   :id: "getting-started"
   :class: my-panel
 
-  Procedures for upgrading to a new version of Scylla Enterprise.
+  Procedures for upgrading to a new version of ScyllaDB Enterprise.
 
   Patch Release Upgrade
 
   * :doc:`Upgrade Guide - ScyllaDB Enterprise 2022.x </upgrade/upgrade-enterprise/upgrade-guide-from-2022.x.y-to-2022.x.z/index>`
-  * :doc:`Upgrade Guide - Scylla Enterprise 2021.x <upgrade-guide-from-2021.x.y-to-2021.x.z/index>`
-  * :doc:`Upgrade Guide - Scylla Enterprise 2020.x <upgrade-guide-from-2020.x.y-to-2020.x.z/index>`
-  * :doc:`Upgrade Guide - Scylla Enterprise 2019.x <upgrade-guide-from-2019.x.y-to-2019.x.z/index>`
-  * :doc:`Upgrade Guide - Scylla Enterprise 2018.x <upgrade-guide-from-2018.x.y-to-2018.x.z/index>`
-  * :doc:`Upgrade Guide - Scylla Enterprise 2017.x <upgrade-guide-from-2017.x.y-to-2017.x.z/index>`
+  * :doc:`Upgrade Guide - ScyllaDB Enterprise 2021.x <upgrade-guide-from-2021.x.y-to-2021.x.z/index>`
+  * :doc:`Upgrade Guide - ScyllaDB Enterprise 2020.x <upgrade-guide-from-2020.x.y-to-2020.x.z/index>`
+  * :doc:`Upgrade Guide - ScyllaDB Enterprise 2019.x <upgrade-guide-from-2019.x.y-to-2019.x.z/index>`
+  * :doc:`Upgrade Guide - ScyllaDB Enterprise 2018.x <upgrade-guide-from-2018.x.y-to-2018.x.z/index>`
+  * :doc:`Upgrade Guide - ScyllaDB Enterprise 2017.x <upgrade-guide-from-2017.x.y-to-2017.x.z/index>`
 
   Major Release Upgrade
 
-  * :doc:`Upgrade Guide - From Scylla Enterprise 2021.1 to Scylla Enterprise 2022.1 <upgrade-guide-from-2021.1-to-2022.1/index>`
-  * :doc:`Upgrade Guide - From Scylla Enterprise 2020.1 to Scylla Enterprise 2021.1 <upgrade-guide-from-2020.1-to-2021.1/index>`
-  * :doc:`Upgrade Guide - From Scylla Enterprise 2019.1 to Scylla Enterprise 2020.1 <upgrade-guide-from-2019.1-to-2020.1/index>`
-  * :doc:`Upgrade Guide - From Scylla Enterprise 2018.1 to Scylla Enterprise 2019.1 <upgrade-guide-from-2018.1-to-2019.1/index>`
-  * :doc:`Upgrade Guide - From Scylla Enterprise 2017.1 to Scylla Enterprise 2018.1 <upgrade-guide-from-2017.1-to-2018.1/index>`
+  * :doc:`Upgrade Guide - From ScyllaDB Enterprise 2022.1 to Scylla Enterprise 2022.2 (minor release) <upgrade-guide-from-2022.1-to-2022.2/index>`
+  * :doc:`Upgrade Guide - From ScyllaDB Enterprise 2021.1 to Scylla Enterprise 2022.1 <upgrade-guide-from-2021.1-to-2022.1/index>`
+  * :doc:`Upgrade Guide - From ScyllaDB Enterprise 2020.1 to Scylla Enterprise 2021.1 <upgrade-guide-from-2020.1-to-2021.1/index>`
+  * :doc:`Upgrade Guide - From ScyllaDB Enterprise 2019.1 to Scylla Enterprise 2020.1 <upgrade-guide-from-2019.1-to-2020.1/index>`
+  * :doc:`Upgrade Guide - From ScyllaDB Enterprise 2018.1 to Scylla Enterprise 2019.1 <upgrade-guide-from-2018.1-to-2019.1/index>`
+  * :doc:`Upgrade Guide - From ScyllaDB Enterprise 2017.1 to Scylla Enterprise 2018.1 <upgrade-guide-from-2017.1-to-2018.1/index>`
```

Comment:
are these still supported?
AFAIU they are not, so if they are not, we should remove from here